data/movie/Bound (1996)/criterion-4k/disc01-summary.txt

```diff
@@ -0,0 +1,34 @@
+Name: Bound (1996)
+Source file name: 00005.mpls
+Type: MainMovie
+Year: 1996
+Duration: 1:48:59
+Chapters count: 20
+Size: 78.7 GB
+Segment count: 1
+Segment map: 9
+AudioTrack[2]: Commentary from Writer/Directors Lana Wachowski & Lilly Wachowski, editor Zach Staenberg, technical consultant Susie Bright. And actors Joe Pantoliano, Jennifer Tilly, and Gina Gershon
+Chapters:
+- Bound
+- Plumbing
+- The Watering Hole
+- Violet's earring
+- Enter Caesar
+- "I had to see you"
+- "We're not that different"
+- Shelly's comeuppance
+- "I want out"
+- The money
+- Their plan
+- "I gotta get the money back"
+- Gino and Johnnie Marzzone
+- Cleaning up
+- Searching Johnnie's place
+- Redial
+- Caesar catches on
+- Mickey returns
+- Spilled paint
+- Happy ending
+- Color bars
+File name: BOUND_t01.mkv
+Description: This commentary track, recorded in 1997, features directors Lana Wachowski and Lilly Wachowski, editor Zach Staenberg, and technical consultant Susie Bright. They are joined later in the track by actors Joe Pantoliano, Jennifer Tilly, and Gina Gershon. To listen to the commentary while viewing the movie, press the AUDIO key on your remote at any time.
```

Comment:
The Description has to go before File name. File name must be the last item in grouping of properties. Once you move it, you will have to re-finalize the relaease